docs/source/tutorials/code_snippets/event_loop_wx.py

```diff
@@ -0,0 +1,10 @@
+from traits.api import HasTraits, Int
+import wx
```

Comment:
Should we follow the usual import order and put 3rd party imports before ETS imports?